core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -2042,6 +2042,9 @@ private[zio] trait ZIOFunctions extends Serializable {
    */
   final def fail[E](error: E): IO[E, Nothing] = haltWith(trace => Cause.Traced(Cause.Fail(error), trace()))
 
+  private[zio] final def failSilently[E](error: E): IO[E, Nothing] =
```

Comment:
How about the following method on ZIO instead:

```scala
final def silence: ZIO[R, E, A] = mapErrorCause(cause => Cause.silent(cause))
```

Then in `Schedule` you could do:

```scala
ZIO.fail(()).silent
```

This would be a bit more compositional and at that point I think you could expose it as a pubic method which would address @neko-kai's earlier point of wanting to make `forkInternal` public.

Reply:
@adamgfraser Great idea!